.github/workflows/test.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         exclude:
         # https://github.com/ruby/setup-ruby/issues/496
         - { os: ubuntu-22.04, ruby: '2.2' }
+        # - { os: ubuntu-24.04, ruby: '2.2' }
```

Comment:
Could you either remove this line if unneeded or if the build does fail then uncomment it?